<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsTab</class>
 <widget class="QWidget" name="StatsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="lay" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="stats_layout">
     <item>
      <widget class="QLabel" name="total_label">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="correct_label">
       <property name="text">
        <string>Correct</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="skipped_label">
       <property name="text">
        <string>Skipped</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="incorrect_label">
       <property name="text">
        <string>Incorrect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <spacer name="button_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="text">
        <string>Главное меню</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
